### ISSUE TITLE
[Bug]: "In Progress" Series Filter issue?

### ISSUE THREAD

**Issue by some-guy-23** [advplyr/audiobookshelf#3220]
### What happened?

When on the Series page, when I use the drop-down filter and select "Series Progress -> in-progress", I only see the series of the book I'm currently listening to (1 result).

### What did you expect to happen?

I'd expect to see a full list of all series where I've listened to at least 1 book in that series (but not completed the series yet). So that I can pick what series to continue next.

This is basically an alternative once the "Continue Series" line overflows the Home page (so can't see all in-progress / continue series).

### Steps to reproduce the issue

1. Go to Series page
2. Select filter Series Progress -> in-progress

### Audiobookshelf version

 v2.11.0

### How are you running audiobookshelf?

Windows Tray App

### What OS is your Audiobookshelf server hosted from?

Windows

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

Chrome

**Comment by some-guy-23** [comment]
After submitting this, occurred to me maybe this is the intention of the "Not Finished" filter behavior? However, for the "Not Finished" filter, it also shows all series that haven't been started yet.

So, there doesn't seem to currently be a way to view all series that are "Started but not Finished" (which I'd think is a synonymous with "In Progress").

**Comment by sydlexius** [comment]
Related to this, when I'm on the "In-progress" filter for series, it shows series where all items are 100% read.  As there's also a "Finished" filter under "Series Progress", seeing finished series under "In-Progress" just creates more noise, IMO.

As for the behavior that you're experiencing, I can't reproduce this issue on my instance (docker, v2.11.0).  I see multiple series when selecting the same filter.

**Comment by sydlexius** [comment]
I don't see any series that have any books without progress.  Everything has one or more partially- or fully-complete books.

**Comment by some-guy-23** [comment]
Yeah so that's my root issue, I have quite a few (unfinished) series with totally completed books, but none of them show up. 

I'm wondering if it's related to having marked the books as completed, as opposed to actually listening to them (listened to them before I used ABS).

**Comment by advplyr** [comment]
This is how it was designed because it matches the library filters available for progress so the query is the same.

I understand what you mean and why it would make sense to interpret it that way.

**Comment by sir-wilhelm** [comment]
I recently ran into this. Initially I thought the same thing, but the current `In Progress` filter does makes sense/have a use case -- show series you are actively listening to.

There is another use case for showing all series that have been started, and I also see how the two could be mixed up. I have some changes ready to add a `Started` filter, so you can see all series that have progress, but are not complete. This would include non-finished series that have completed books (with or w/o progress). It would help show started Series that have been removed from `Continue Listening`.
